src/main/java/com/nerdtranslator/lingueeapibridge/service/impl/NaturalLangServiceImpl.java

```diff
@@ -0,0 +1,98 @@
+package com.nerdtranslator.lingueeapibridge.service.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.language.v1.*;
+import com.nerdtranslator.lingueeapibridge.service.NaturalLangService;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Properties;
+
+@Service
+public class NaturalLangServiceImpl implements NaturalLangService {
+
+    @Override
+    public String analyzeText(String textToAnalyze) {
+        return analyzeTextMethod(textToAnalyze);
+    }
+
+    private static Map<String, Object> getCredentials() {
+        Properties properties = new Properties();
+        Map<String, Object> credentials = new LinkedHashMap<>();
+
+        try (FileInputStream inputStream = new FileInputStream("src/main/resources/sensitive.properties")) {
```

Comment:
Don't read credentials straight from sensitive.properties, add variables as link in application.properties, inject them using Value annotation